man/systemd.exec.xml

```diff
@@ -225,6 +225,53 @@
         <xi:include href="system-only.xml" xpointer="singular"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ProcMode=</varname></term>
+
+        <listitem><para>Takes one of <literal>noaccess</literal>, <literal>invisible</literal>,
+        <literal>ptraceable</literal> or <literal>default</literal>. Defaults to the latter. This controls
+        the <literal>hidepid=</literal> mount option of the <literal>procfs</literal> instance for the unit:
+        when set to <literal>noaccess</literal> the ability to access most of other user's process metadata
+        in <filename>/proc/</filename> is taken away for processes of the service. When set to
+        <literal>invisible</literal> processes owned by other users are hidden from
+        <filename>/proc/</filename>. If <literal>ptraceable</literal> all processed that cannot be
+        <function>ptrace()</function>'ed by a process are hidden to it. If <literal>default</literal> no
+        restrictions on <filename>/proc/</filename> access or visibility are made. For further details see
+        <ulink url="https://www.kernel.org/doc/Documentation/filesystems/proc.rst">proc.rst</ulink>. It is
+        generally recommended to run most system services with this option enabled, if possible. This option
```

Comment:
I'd drop "if possible" — it's a recommendation, so it is clear that that it doesn't always apply.